learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+- [ ] We can write functions that add DOM nodes
+- [ ] We can write functions that remove DOM nodes
+- [ ] We can write functions that modify DOM nodes
+
+### Event listeners
+
+- [ ] **We can apply click event listeners to buttons**
```

Comment:
This might be too specific? They should probably be using a form `"submit"` event for the project this week for example